src/components/home.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component, PropTypes } from 'react';
+
+const BASE_URL = "https://dev-570863.oktapreview.com";
```

Comment:
Might be an optimization for later, but will probably want to move these out to a config file and have it get populated by the express server on startup. I.e. if I'm a user hooking this up to my prod account, I need to modify these config items.